src/main/kotlin/com/quipalup/katydid/logentry/secondaryadapter/database/LogEntryDatabase.kt

```diff
@@ -17,8 +16,8 @@ import org.springframework.data.repository.findByIdOrNull
 
 @Named
 class LogEntryDatabase(private val jpaLogEntryRepository: JpaLogEntryRepository) : LogEntryRepository {
-    override fun create(logEntry: LogEntry): Either<CreateLogEntryError, Id> = logEntry.toJpa()
-        .flatMap { jpaLogEntry: JpaLogEntry -> jpaLogEntry.ensureDoesNotExist() }
+    override fun save(logEntry: LogEntry): Either<SaveLogEntryError, Id> = logEntry.toJpa()
+//        .flatMap { jpaLogEntry: JpaLogEntry -> jpaLogEntry.ensureDoesNotExist() }
```

Comment:
I guess you will uncomment this line over here?

Reply:
Yes, I have to still change things around here

Reply:
Hello, I reviewed again your PR, this line is not necessary in this layer. 

Let's move that logic to the application layer. Please see the comment below